### PR TITLE
feat: Qdrant vector store node

### PR DESCRIPTION
**Pull request by Anush008**
Hello folks 👋. Greetings from [Qdrant](https://qdrant.tech/).

## Description

This PR intends to add support for Qdrant to be used as a vector store in Rivet.

This implementation adds no additional dependencies.

### CODE REVIEW

File: packages/core/src/plugins/qdrant/QdrantVectorDatabase.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  constructor(settings: Settings) {
+    this.#apiKey = settings.pluginSettings?.qdrant?.qdrantApiKey as string | undefined;
+    this.#qdrantUrl = settings.pluginSettings?.qdrant?.qdrantUrl as string | undefined;
+  }
+
+  async store(collection: DataValue, vector: VectorDataValue, data: DataValue, { id }: { id?: string }): Promise<void> {
+    const collectionName = coerceType(collection, 'string');
+
+    if (!id) {
+      id = crypto.randomUUID();
```
Comment (Anush008) [inline packages/core/src/plugins/qdrant/QdrantVectorDatabase.ts:25]: To avoid add a separate `uuid` dependency, we use Node's `crypto` module

**Comment by abrenneke** [comment]
Thank you @Anush008! The Vector Store / Vector KNN nodes are actually kind of deprecated, and these days I advise people to write full plugins with dedicated nodes instead, what do you think about doing that? It gives you a lot more flexibility to integrate more deeply with whatever system you're bringing in. The Vector Store and Vector KNN were some of the first integrations I wrote, and I found out that trying to support the lowest-common-demoninator of all vector store implementations wasn't a good idea.

**Comment by Anush008** [comment]
Hi @abrenneke. Thanks for letting me know. Will look into doing that.

**Comment by abrenneke** [comment]
Any update @Anush008? Would love to get another plugin in to Rivet!

**Comment by Anush008** [comment]
Hi @abrenneke. The plugin is done. We plan to publish the package Monday. Will update this PR to reflect the package.

**Comment by abrenneke** [comment]
that's awesome to hear!

**Comment by Anush008** [comment]
Hey @abrenneke. Was caught up with some work last week.
Could you please take a look at https://github.com/Anush008/rivet-plugin-qdrant?
There are some more point operation nodes to add. We'll then move the repo to Qdrant and publish the package this week.

**Comment by Anush008** [comment]
Hey @abrenneke. Was caught up with other parts at work last week.
Could you please skim through  https://github.com/Anush008/rivet-plugin-qdrant?
We'll then move the repo to Qdrant and publish the package this week.

There are some more point operation nodes I plan to add. 

I've updated this PR to reference the external package.

**Comment by abrenneke** [comment]
Looks great @Anush008! Update this PR to add your plugins to `plugins.ts` and let's get this in!

**Comment by Anush008** [comment]
`plugins.ts` should be updated.

**Review by abrenneke — APPROVED** [review]
Awesome!

**Comment by Anush008** [comment]
Hi @abrenneke. Is there any timeline for the next release?
We can publish the integration docs and social posts at the same time.

**Comment by Anush008** [comment]
Also FYI, we've moved the repo and published the package.

**Comment by abrenneke** [comment]
Drafting up a release now @Anush008 sorry for the delay!

**Pull request merged**